src/components/UniversitySearch.js

```diff
@@ -80,7 +80,7 @@ function UniversitySearch() {
                   component="th"
                   scope="row"
                   style={{ fontSize: 15 }}
-                  to={`/get_uni/${uni.name}`}
+                  to={`/get_uni/${id + 1}/0`}
```

Comment:
Is there a reason there is a 0 after it?

Reply:
yes, because now the get_uni url takes in the university id and a param for which info it's showing (0 = discussion, 1 = previously approved courses, 2 = discussion)

Reply:
yea i thought so... one issue i noticed is that the url doesnt change when you switch tabs

Reply:
true, i noticed this too, i'll try to change this now

Reply:
done